pom.xml

```diff
@@ -143,7 +143,7 @@
             <plugin>
                 <groupId>org.jenkins-ci.tools</groupId>
                 <artifactId>maven-hpi-plugin</artifactId>
-                <version>3.5</version>
+                <version>3.17</version>
```

Comment:
Would you like to prepare this as a separate PR?

Reply:
I'll open a separate PR for this and likely point to a later version than that mentioned above